crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -721,7 +721,7 @@ define_tedge_config! {
         lock_files: bool,
 
         /// Interval at which the memory usage is logged (in seconds)
```

Comment:
Perhaps writing what interval 0 means (like "0 means disabled.") is more helpful in this line, but it's ok as it is

Reply:
resolved by aecec18f6e815aab20f00cb0c1bdecf6ebbd97fc